### DISCUSSION TITLE
Didn't get a response from GitHub education for my request that I have submitted before

### DISCUSSION THREAD

**Discussion by Promenak** [community/community#19624] [GitHub Education] [answered]
I submitted a request on June 25 and got an email saying that I should get a response from GitHub education by June 28 but I haven't received anything from GitHub education yet
shall I cancel my request and submit a new one or just wait until they notice it being there?

**Comment by ApidriuC** [comment]
Just wait

**Comment by melvincwng** [comment, marked as answer]
@Un-known-User, don't cancel your current application.

Kindly wait for your current application to be approved. It may take longer than expected.

If you wish to, you can raise a ticket & check with [GitHub Education Support](https://support.github.com/contact/education) over here on the status of your application.

Hope this is helpful, if it is, kindly 'mark as answer' so that the rest are aware this issue has been addressed.

Take care, and have a good day 😄
